test/lib/ufe/testParentCmd.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from ufeTestUtils import mayaUtils, usdUtils
+from ufeTestUtils.testUtils import assertVectorAlmostEqual
+
+import ufe
+
+import maya.cmds as cmds
+
+import unittest
+import os
+
+def childrenNames(children):
+    return [str(child.path().back()) for child in children]
+
+def matrixToList(m):
+    mList = []
+    for i in m.matrix:
+        mList = mList + i
+    return mList
+
+class ParentCmdTestCase(unittest.TestCase):
+    '''Verify the Maya parent command on a USD scene.'''
+
+    pluginsLoaded = False
+    
+    @classmethod
+    def setUpClass(cls):
+        if not cls.pluginsLoaded:
+            cls.pluginsLoaded = mayaUtils.isMayaUsdPluginLoaded()
+    
+    @classmethod
+    def tearDownClass(cls):
+        cmds.file(new=True, force=True)
+
+    def setUp(self):
+        # Load plugins
+        self.assertTrue(self.pluginsLoaded)
+
+        # Load a file that has the same scene in both the Maya Dag
+        # hierarchy and the USD hierarchy.
+        filePath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "test-samples", "parentCmd", "simpleSceneMayaPlusUSD_TRS.ma" )
+        cmds.file(filePath, force=True, open=True)
+
+        # Clear selection to start off
+        cmds.select(clear=True)
+
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 115, 'Parent functionality only available in Maya Preview Release 115 or later.') 
+    def testParentRelative(self):
```

Comment:
Test parent -relative.  This is the simpler case, as the object's local transformation is unaffected.